src/ssl.c

```diff
@@ -14072,6 +14072,22 @@ static WC_INLINE void RestoreSession(WOLFSSL* ssl, WOLFSSL_SESSION* session,
 #endif
 }
 
+static int SslSessionCacheOn(const WOLFSSL* ssl, const WOLFSSL_SESSION* session)
+{
+    (void)session;
+    if (ssl->options.sessionCacheOff
+    #if defined(HAVE_SESSION_TICKET) && defined(WOLFSSL_FORCE_CACHE_ON_TICKET)
```

Comment:
Also to clarify, this is the case where session tickets is enabled at build-time, but the handshake did not use session tickets? Is this for the TLS v1.2 case with classic session resumption based on session id and session cache?

Reply:
If there is a session ticket to cache, OpenSSL always caches the session ticket. The "ID style" session cache options are governed by `ssl->options.sessionCacheOff`.